library/src/main/java/com/novoda/downloadmanager/lib/DownloadThread.java

```diff
@@ -673,6 +677,20 @@ private void handleEndOfStream(State state) throws StopRequestException {
         }
     }
 
+    private void pause() throws StopRequestException {
```

Comment:
@xrigau I checked with @ouchadam the batch will be paused when the individual download is paused.